core/controllers/question_editor_test.py

```diff
@@ -364,10 +305,28 @@ def test_put_with_admin_email_allows_updation(self):
         self.login(self.ADMIN_EMAIL)
         csrf_token = self.get_new_csrf_token()
         _ = self.put_json(
-            '%s/%s/%s' % (
-                feconf.QUESTION_SKILL_LINK_URL_PREFIX, self.question_id,
-                self.skill_id
-            ), {'new_difficulty': 0.9}, csrf_token=csrf_token)
+            '%s/%s' % (
+                feconf.QUESTION_SKILL_LINK_URL_PREFIX, self.question_id
+            ), {
+                'new_difficulty': 0.9,
+                'action': 'update_difficulty',
+                'skill_id': self.skill_id
+            }, csrf_token=csrf_token)
+
+        _ = self.put_json(
```

Comment:
Here and elsewhere (above and below), you can omit the `_ =`

Reply:
Done